apps/agentic-server/index.ts

```diff
@@ -2,214 +2,21 @@ import dotenv from 'dotenv';
-              'The amount of native asset to send along with the Tx, in precision'
-            ),
-          chainId: z
-            .number()
-            .describe('The chain ID to send the transaction on'),
-        }),
-      },
-      executeSwap: {
-        description:
-          'Sends a transaction which executes the swap the user has confirmed.',
-        parameters: z.object({}),
-      },
-      searchTokens: {
-        description: `
-        Search for tokens by name or symbol
-        Returns tokens matching the search term, sorted by 7-day USD volume.
-        For args parsing, use text proximity to map user input to the correct network.
-        `,
-        parameters: z.object({
-          searchTerm: z.string().describe('The search term to find tokens'),
-          network: NetworkEnum.optional().describe('The network to search on'),
-        }),
-      },
-      bebopRate: {
-        description: `Fetches a swap rate from Bebop and displays it to the user.
-
-        Returns an object with the following fields, for display to the user
-        - sellAmountCryptoPrecision: The sell amount in precision format
-        - buyAmountCryptoPrecision: The buy amount in precision format
-        - buyAsset: Object describing the buy asset (i.e symbol, decimals, name, address)
-        - sellAsset: Object describing the sell asset (i.e symbol, decimals, name, address)
-        - approvalTarget: The address the funds will be spent to. Use this as the spender for allowance checks.
+  const { message, id } = req.body;
 
-        **Instructions for LLM:**
-        - Do not display base unit values, feeData, rate, swapperName, asset objects, allowanceTarget, or quote to the user unless specifically asked for technical details.
-        - If the user requests technical details, you may display internal fields.`,
-        parameters: z.object({
-          chain: z
-            .string()
-            .describe('Chain name, e.g. ethereum, arbitrum, polygon, etc.'),
-          fromAsset: z
-            .object({
-              address: z.string(),
-              decimals: z.number(),
-              symbol: z.string().optional(),
-            })
-            .describe('Asset to sell'),
-          toAsset: z
-            .object({
-              address: z.string(),
-              decimals: z.number(),
-              symbol: z.string(),
-            })
-            .describe('Asset to buy'),
-          sellAmountCryptoPrecision: z
-            .string()
-            .describe('Amount to sell in human format, e.g. 1 for 1 ETH'),
-          fromAddress: z
-            .string()
-            .describe(
-              'The address the user is swapping from. Also referred to as "sell address", and should be gotten using the getAddress() tool beforehand'
-            ),
-        }),
-      },
-    },
+  const agent = mastra.getAgent('shapeshiftAgent');
+  const result = await agent.stream(message, {
+    resourceId: 'user',
+    threadId: id,
```

Comment:
The only real change in this PR - moves the agent to `shapeShiftAgent` mastra agent and consumes it here - underlying logic doesn't change